### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by bpas247**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,4034 +1,3988 @@
...
-            "version": "1.0.2",
-            "resolved": "https://registry.npmjs.org/is-descriptor/-/is-descriptor-1.0.2.tgz",
-            "integrity": "sha512-2eis5WqQGV7peooDyLmNEPUrps9+SXX5c9pL3xEB+4e9HnGuDa7mB7kHxHw4CbqS9k1T2hOH3miL8n8WtiYVtg==",
-            "requires": {
-              "is-accessor-descriptor": "^1.0.0",
-              "is-data-descriptor": "^1.0.0",
-              "kind-of": "^6.0.2"
-            }
+        },
+        "e
```
Comment (etcadinfinitum) [inline package-lock.json]: If we have `tar` as a primary dependency, will that module get used for the requirements of other packages? Sorry for the n00b question.

Comment (etcadinfinitum) [inline package-lock.json]: Version looks good; I am going to _assume_ that the dependency versions are correct. (Side note: I assume some, or all, of this is auto-generated?)

Comment (etcadinfinitum) [inline package-lock.json]: Is this correct?

Comment (etcadinfinitum) [reply package-lock.json]: This preview sucks: link to code comment is https://github.com/UWB-ACM/uwbacm-site/pull/15/files#r286721614

Comment (etcadinfinitum) [reply package-lock.json]: This preview sucks: link to code comment is https://github.com/UWB-ACM/uwbacm-site/pull/15/files#r286721863

Comment (etcadinfinitum) [reply package-lock.json]: This preview sucks; link to code comment is https://github.com/UWB-ACM/uwbacm-site/pull/15/files#r286722106

File: package.json [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
-      "url": "https://github.com/uwb-acm/uwbacm-site/issues"
-    },
-    "homepage": "https://github.com/uwb-acm/uwbacm-site#readme",
-    "dependencies": {
-      "gulp": "^4.0.0",
-      "gulp-sass": "^4.0.2",
-      "sass": "^1.17.0"
-    }
+  "name": "uwbacm-site",
+  "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (bpas247) [reply package.json]: > Are we actually using versioning?

Afaik we have never used versioning for this website. We should open an issue about that :eyes: 

Also this diff doesn't actually add anything, I guess I re-formatted the file before pushing. 
I'm gonna revert the changes for this file :+1:

File: package-lock.json (hunk 1 above)
Comment (bpas247) [reply package-lock.json]: > Side note: I assume some, or all, of this is auto-generated?

This is all auto-generated via [npm update](https://docs.npmjs.com/cli/update.html)

Comment (etcadinfinitum) [reply package-lock.json]: ok cool, thank you for clarifying

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (etcadinfinitum) [reply package.json]: word. no big deal

**Pull request merged**